### DISCUSSION TITLE
How to install Vorta on Debian 10 ?

### DISCUSSION THREAD

**Discussion by Jason-MK** [borgbase/vorta#1061] [General]
Hello, 

Can anyone tell me or give me a documentation that works to how to install Vorta on Debian on my local machine ? 

And how to create a local repository on the same Debian machine ? 

I tried to install Vorta but I couldn't I'm using VMware host on my local machine and I want to know to access Vorta and how to install it please ?

Best Regards

**Comment by m3nu** [comment]
Personally, I'd use PIP. Doing this regularly in VMs for testing.

https://vorta.borgbase.com/install/linux/#install-from-source

Some people prefer Flatpak because it deals with deps and sets up the app icon.

**Comment by bastiencyr** [comment]
Hello @Jason-MK , you can also find vorta in the apt repository in Debian 11 ! Debian 11 was released recently 🤗

**Reply by sten0** [reply]
@Jason-MK if upgrading to Debian 11 (bullseye) is not possible at this time, please note that buster-backports (Debian 10) has the same version as bullseye.
